__tests__/endpoint.test.js

```diff
@@ -3,26 +3,41 @@ const request = require('supertest')
 const seed = require('../db/seeds/seed')
 const app = require('../app/app')
 const db = require('../db/connection')
+const endpoint = require('../endpoints.json')
 
 beforeEach(() => seed(data));
 afterAll(() => db.end());
 
 describe('Endpoint Tests', () => {
-    describe.only('/api/topics', () => {
+    describe('/api/topics', () => {
         test('200: get 200 response', () => {
             return request(app).get('/api/topics').expect(200)
         })
         test('200: returns all data from topics when requesting all data', () => {
             return request(app).get('/api/topics').expect(200).then(({body}) => {
                 body.data.forEach(topics =>{
-                    expect(typeof topics.slug). toBe('string')
+                    expect(typeof topics.slug).toBe('string')
                     expect(typeof topics.description).toBe('string')
                 })
             })
         })
-        test.only('404: get 404 response', () => {
+        test('404: get 404 response', () => {
             return request(app).get('/api/topiks').expect(404).then(({body}) => {
-                console.log(body)
+                expect(body.msg).toBe(undefined)
+            })
+        })
+    })
+    describe('/api', () => {
+        test('200: get 200 response', () => {
+            return request(app).get('/api').expect(200)
+        })
+        test('200: returns all data from endpoints.json when requesting all data', () => {
+            return request(app).get('/api').expect(200).then(({body}) => {
+                expect(body.data).toEqual(endpoint)
+            })
+        })
+        test('404: get 404 response', () => {
+            return request(app).get('/apl').expect(404).then(({body}) => {
```

Comment:
for feedback on this test, please see my comment re. error handling on your previous task.